identify/extensions.py

```diff
@@ -45,6 +45,7 @@
     'ear': {'binary', 'zip', 'jar'},
     'edn': {'text', 'clojure', 'edn'},
     'ejs': {'text', 'ejs'},
+    'env': {'text', 'env', 'dotenv'},
```

Comment:
should be one or the other, not both

Reply:
Alright, removed env and kept dotenv (and replaced the commit)